jumpscale/core/identity/__init__.py

```diff
@@ -87,6 +98,13 @@ def explorer(self):
             from jumpscale.clients.explorer import export_module_as  # Import here to avoid circular imports
 
             ex_factory = export_module_as()
+
+            # Backward compitablity (Update mainnet url)
+            # Create new config has_migrated = True after mainnet update
+            if js_config.get("has_migrated_explorer_url", False):
+                if urlparse(self.explorer_url).hostname == urlparse(EXPLORER_URLS["mainnet"]).hostname:
+                    self.explorer_url = EXPLORER_URLS["mainnet"]
```

Comment:
We should set `has_migrated_explorer_url` to `True` here, so we don't do it again.